packages/quick-edit/package.json

```diff
@@ -10,12 +10,13 @@
 	"license": "BSD-3-Clause",
 	"author": "workers-devprod@cloudflare.com",
 	"scripts": {
-		"build": "./build.sh",
+		"custom:build": "./build.sh",
```

Comment:
Is this change reflected elsewhere? Are there places that expect this script to be called `build`?

Reply:
I can go over that with @penalosa however this is to prevent this build from mistakenly getting picked up by TurboRepo when we don't want this to actually build like the rest of the monorepo, it is a special case.

Reply:
Is this the one with the 1hr+ build time that runs separately?